src/Ens/JobeetBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Ens\JobeetBundle\Tests\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class DefaultControllerTest extends WebTestCase
+{
+    public function testIndex()
+    {
+        $client = static::createClient();
+
+        $crawler = $client->request('GET', '/hello/Fabien');
+
+        $this->assertTrue($crawler->filter('html:contains("Hello Fabien")')->count() > 0);
```

Comment:
@PhotonSapiens 
wow, you even write tests
One notice however - please use annotations for tests, i.e. in this case:
```PHP
/**
 * @test
*/
indexShouldGreet()
{
    // same body here
}
```